examples/visualization/plot_contour.ipynb

```diff
@@ -74,8 +74,7 @@
     "\n",
     "def objective(trial):\n",
     "    model = L.Classifier(create_model(trial))\n",
-    "    optimizer = chainer.optimizers.Adam()\n",
-    "    optimizer.setup(model)\n",
+    "    optimizer = chainer.optimizers.Adam().setup(model)\n",
```

Comment:
This is a detail so please feel free to ignore it but I believe it's quite rare to chain `setup` this way. It's perfectly valid.

Reply:
Yes, I've been pushing for this shorter form since I worked to get the PRs done to enable it over a year ago. Denser is better. ;)